src/AppContext/Infrastructure/Repository/DoctrineRepositories.php

```diff
@@ -83,6 +83,21 @@ public function find(string $id, $mode = self::MODE_FETCH_NOTHING): Entity\Repos
         return $repository;
     }
 
+    public function findByIdentifier(string $type, string $identifier): Entity\Repository
+    {
+        /** @var Entity\Repository|null $repository */
+        $repository = $this->repo()->findOneBy([
+            'type' => $type,
+            'identifier' => $identifier,
+        ]);
+
+        if ($repository === null) {
+            throw Repository\Exception\NotFound::forIdentifier($identifier);
```

Comment:
Avoid using static access to class '\Regis\AppContext\Domain\Repository\Exception\NotFound' in method 'findByIdentifier'.